enterprise/coderd/schedule/template_test.go

```diff
@@ -297,7 +298,7 @@ func TestTemplateUpdateBuildDeadlines(t *testing.T) {
 				FailureTTL:               0,
 				TimeTilDormant:           0,
 				TimeTilDormantAutoDelete: 0,
-			})
+			}, nil, slog.Logger{})
```

Comment:
Please pass a real logger here in case something fails in the tests.

```go
slogtest.Make(t, &slogtest.Options{IgnoreErrors: true}).Leveled(slog.LevelDebug)
```